src/lib/cookies.js

```diff
@@ -42,11 +42,17 @@ const addCookie = (name, value, daysToLive, isCrossSubdomain = false) => {
 	const expires = new Date();
 
 	if (!isValidCookieValue(name) || !isValidCookieValue(value)) {
-		reportError(
-			new Error(`${ERR_INVALID_COOKIE_NAME} .${name}=${value}`),
-			{},
-			false,
-		);
+		if (window.guardian?.modules?.sentry?.reportError) {
```

Comment:
This is the only time we're making this check. DCR goes to some lengths to ensure this function is available and stubbed before Senty is loaded. Is it necessary to also make this check here?